src/Service/ReminderHelper.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace App\Service;
+
+use App\Entity\User;
+use App\Repository\ReminderRepository;
+use DateTime;
+use Doctrine\ORM\EntityManagerInterface;
+
+class ReminderHelper
+{
+    /**
+     * @var EntityManagerInterface
+     */
+    private $entityManager;
+    /**
+     * @var ReminderRepository
+     */
+    private $reminderRepository;
+
+    public function __construct(EntityManagerInterface $entityManager, ReminderRepository $reminderRepository)
+    {
+        $this->entityManager = $entityManager;
+        $this->reminderRepository = $reminderRepository;
+    }
+
+    public function updateStatuses(): bool
```

Comment:
Some documentation about what this method does would help a lot.

Reply:
Added documentation to the `updateStatuses` function.